core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -238,8 +263,10 @@ static String getBrokerEndpoints(List<Broker> brokerList) {
       for(EndPoint ep : JavaConversions.asJavaCollection(broker.endPoints().values())) {
         String connectionString = ep.connectionString();
 
-        if (connectionString.startsWith(SchemaRegistryConfig.KAFKASTORE_SECURITY_PROTOCOL_SSL + "://")
-            || connectionString.startsWith(SchemaRegistryConfig.KAFKASTORE_SECURITY_PROTOCOL_PLAINTEXT + "://")) {
+        if (connectionString.startsWith(SecurityProtocol.PLAINTEXT.toString() + "://")
+            || connectionString.startsWith(SecurityProtocol.SSL.toString() + "://")
+            || connectionString.startsWith(SecurityProtocol.SASL_PLAINTEXT.toString() + "://")
+            || connectionString.startsWith(SecurityProtocol.SASL_SSL.toString() + "://")) {
           if (sb.length() > 0) {
             sb.append(",");
           }
```

Comment:
Looks like the warning below should be updated?
`           log.warn("Ignoring non-plaintext and non-SSL Kafka endpoint: " + connectionString);`